core/blockchain.go

```diff
@@ -525,7 +525,13 @@ func (bc *BlockChain) Stop() {
 		return
 	}
 
+	log.Info("Shutting down state manager")
+	if err := bc.stateManager.Shutdown(); err != nil {
+		log.Error("Failed to Shutdown state manager", "err", err)
+	}
+
 	// Unsubscribe all subscriptions registered from blockchain.
+	log.Info("Clsoing subscriptions scope")
```

Comment:
```suggestion
	log.Info("Closing subscriptions scope")
```

Reply:
on the second thought if we don't need this can we remove it so it can be consistent with the coreth?